frontend/device/android/powerd.lua

```diff
@@ -3,17 +3,47 @@ local _, android = pcall(require, "android")
 
 local AndroidPowerD = BasePowerD:new{
     fl_min = 0, fl_max = 25,
-    fl_intensity = 10,
+--    fl_intensity = 10,
 }
 
 function AndroidPowerD:frontlightIntensityHW()
-    return math.floor(android.getScreenBrightness() / 255 * self.fl_max)
+    local intensity = math.floor(android.getScreenBrightness() / 255 * self.fl_max)
```

Comment:
the way to calcule intensity can be made generic for future implementations with:

`math.floor(android.getScreenBrightness() / android.getScreenMaxBrightness() * self.fl_max)`

Reply:
Ok. Shall I leave the check `if intensity < 0` for now?